relentless/data.py

```diff
@@ -1,8 +1,23 @@
-"""Core data management.
+"""
+Data management
```

Comment:
I opened issue #71 to remove the `Project` class, which is largely unused. This module then only contains `Directory`. Maybe we should think about combining `data` with `_collections` or something. You can leave this description as-is since we might want to change it later anyway.